compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/error.idx-outlining.js

```diff
@@ -0,0 +1,12 @@
+import idx from 'idx';
```

Comment:
Nit: could we rename this fixture to `error.bug-idx-outlining` to differentiate between bugs and expected errors?

Reply:
oops, landed too quickly! i'm gonna try and fix this bug so hopefully I'll be able to rename it then